### PR TITLE
PIM-5002: how to apply a EE patch

### PR DESCRIPTION
**Pull request by solivier**

### CODE REVIEW

File: cookbook/apply_patch/how_to_apply_a_patch.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+     $ cd /path/to/you/project/directory
+     $ composer update
+
+Then don't forget to clean and rebuild you cache
+
+ .. code-block:: bash
+    :linenos:
+
+     $ rm app/cache/* -rf
+     $ php app/console cache:clear --env=prod --no-debug
```
Comment (jmleroux) [inline cookbook/apply_patch/how_to_apply_a_patch.rst]: Not a duplicate of the line just above ?

Comment (solivier) [inline cookbook/apply_patch/how_to_apply_a_patch.rst]: no as it rebuilds the cache, we do the same in migration readme

**Comment by jmleroux** [comment]
GTM

**Comment by jjanvier** [comment]
gtm

**Pull request merged**